src/components/LandingLabAbout/index.tsx

```diff
@@ -73,11 +80,19 @@ export const LandingLabAbout: FC = () => {
                   {user?.public_repos || 182}
                 </strong>{" "}
                 <span className='w-full'>Open Source Projekte</span>
-                <strong
-                  className={["px-4 py-3 bg-purple text-white"].join(" ")}
-                >
-                  <GitHub />
-                </strong>
+                <Link href='https://github.com/technologiestiftung'>
```

Comment:
Don't use next.js Link component for external links, only for internal links. The Link component does things like prerendering, routing etc. Nothing relevant for external links.

Reply:
Also, very cool that you added the link! I would prefer to have the whole area clickable and not only the github symbol.